app/components/task/details/output/task-log.component.ts

```diff
@@ -161,11 +161,21 @@ export class TaskLogComponent implements OnInit, OnChanges, OnDestroy {
             this.fileLoaderMap[filename] = fileLoader;
 
             if (this._shouldGetFileSize(filename)) {
-                fileLoader.getProperties().subscribe((file: File) => {
-                    if (file) {
-                        const props = file.properties;
-                        this.fileSizes[filename] = prettyBytes(props && props.contentLength);
-                    }
+                fileLoader.getProperties().subscribe({
+                    next: (file: File) => {
+                        if (file) {
+                            const props = file.properties;
+                            this.fileSizes[filename] = prettyBytes(props && props.contentLength);
+                        }
+                    },
+                    error: (error) => {
```

Comment:
Because it just ignore the log error it doesn't actually ignore the error.

If you want to ignore the error you'll have to do .catch but I don't think that's a good idea